### PR TITLE
Add description for `--package-paths` option into readme.md

### PR DESCRIPTION
**Pull request by kitwtnb**
I added explanation to `Readme.md` as I have to look at #157 to find out the options when using multiple `Package.swift`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,11 @@ You can see options by `license-plist --help`.
 - Default: `Package.swift`
 - `LicensePlist` tries to find `YourProjectName.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved` and `YourProjectName.xcworkspace/xcshareddata/swiftpm/Package.resolved`, then uses new one.
 
+#### `--package-paths`
+
+- Support for multiple `Pakckage.swift`
```
Comment (timtimht) [inline README.md:118]: Typo Pakckage

Comment (kitwtnb) [reply README.md]: thanks, fixed it.

**Comment by mono0926** [comment]
Thanks, released: https://github.com/mono0926/LicensePlist/releases/tag/3.22.2

**Comment by treastrain** [comment]
Thank you for making up for my lack of consideration! @kitwtnb

**Pull request merged**